packages/peregrine/lib/talons/Checkout/usePaymentsFormItems.js

```diff
@@ -0,0 +1,88 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+// TODO install informed?
```

Comment:
@jimbo I want your thoughts on this talon.

Reply:
Yes, please add `informed` as a peer dep to `peregrine`. We're going to need its hooks.

Reply:
Done.